docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/04_instrument_additional_configurations.md

```diff
@@ -0,0 +1,59 @@
+```xml
+<instrumentPermissions>
+    <useInstrumentPermissions>false</useInstrumentPermissions>
+    <instrument>
+        <Test_name>sampleInstrument</Test_name>
+        <permission>sampleInstrumentPermissionName</permission>
+    </instrument>
+    <instrument>
+        <Test_name>sampleInstrument2</Test_name>
+        <permission>sampleInstrument2PermissionName</permission>
+    </instrument>
+</instrumentPermissions>
+```
+
+> _**NOTE:** Leaving the `useInstrumentPermissions` as the default (`false`) will grant all users having access to the timepoint access to all instruments at this timepoint. Setting the value to `true` will only affect instruments added to an `<instrument>` tag, instruments not specifically mentioned will not be affected._
+
+### Certification
+Instrument certification is a feature by which some instruments can only be completed, in a clinical setting, by a certified examiner. For that reason, LORIS will limit the examiner options available in the instrument to "certified" examiners (For more details about examiners, see [Adding Examiners](TODO) in the Getting Started section). 
+
+In order to use this functionality, the `EnableCertification` flag must first be enabled by changing the `0` to `1` in the `project/config.xml` configuration file. The next step is to complete the `CertificationProjects` section by adding a new `<CertificationProject>` tag for every project using the certification functionality. Finally, a new `<test>` tag should be added for each instrument using this feature under the `<CertificationInstruments>` parent tag. In the example below the instrument certification feature is enabled for projects 1 and 5 and instrument "sampleInstrument".
+
+```
+<Certification>
+	<EnableCertification>1</EnableCertification>
+	<CertificationProjects>
+		<CertificationProject>1</CertificationProject>
+		<CertificationProject>5</CertificationProject>
+	</CertificationProjects>
+	<CertificationInstruments>
+		<test value="sampleInstrument">Sample Instrument Display Name</test>
+	</CertificationInstruments>
+</Certification>
+```
+> _**NOTES:**_
+ 
+> - _The projects listed in the `CertificationProjects ` section are the identifiers of the projects derived from the `Project` table of the database._
+> - _The syntax of the `<test>` tag is important. The tag must have a `value` attribute set to the "testname" found in the `test_names` table of the database. The instrument's display name should be added between the open and the closing tag as seen in the example above._
+### Surveys
+
+Survey Instruments allow participants to enter the data directly in the instrument without going through an examiner or a data-entry personnel. Study coordinators can use the front-end Survey module to create a unique survey key for each instrument for each participant and email the survey to a participant. Survey functionality can be enabled on a participant per participant basis.
```

Comment:
I don't know what "Survey functionality can be enabled on a participant per participant basis" means but I don't think it's true? It's enabled on a per instrument basis..